html_css/v2/main-course/positioning.md

```diff
@@ -0,0 +1,38 @@
+### Positioning
+
+You have learned various ways how to position elements on the screen and even though the methods you know so far are sufficient in most cases, these CSS rules are still very powerful and will certainly help you step up your CSS game!
+
+### Learning Outcomes
+ - You'll learn how to use absolute positioning.
+ - You'll learn how to use fixed positioning.
+ - You'll learn how to use sticky positioning.
+ - You'll know the difference between each property and how to combine them.
+
+### Static and Relative Positioning
+The difference between static and relative is fairly simple. Static is the default position of every element, and properties _top_, _right_, _bottom_, and _left_ have no affect on statically positioned elements. Relative on the other hand is pretty much the same as static, but properties  _top_, _right_, _bottom_, and _left_ affect the specific element relative to itself.
+
+### Absolute Positioning
+Absolute positioning allows you to _absolutely_ position elements on the screen. Absolutely positioned elements will be relative to the next absolute or relative parent. Absolutely positioned elements are also removed from the normal flow of the document. To put it in simple terms: elements that are removed from the normal flow of the document don't affect other elements and are also not affected by other elements. Using absolute positioning allows you to position elements anywhere on the screen using  _top_, _right_, _bottom_, and _left_ properties. This property is really useful when you want to position something at an exact point on the screen, without disturbing any of the other elements.
+
+### Fixed Positioning 
```

Comment:
nothing wrong with this section.. another common example though: those little floating chat-buttons (we have one on TOP, might be worth pointing that out.)